### PR TITLE
adding smoothr

### PR DESCRIPTION
**Pull request by Rassine**
r-smoothr recipe was build thanks to conda skeleton CRAN
I marked miself as maintainer of the recipe

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Rassine** [comment]
@conda-forge/help-r please review this recipe

File: recipes/r-smoothr/build.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+export DISABLE_AUTOBREW=1
+
+# R refuses to build packages that mark themselves as Priority: Recommended
+mv DESCRIPTION DESCRIPTION.old
+grep -va '^Priority: ' DESCRIPTION.old > DESCRIPTION
+# shellcheck disable=SC2086
+${R} CMD INSTALL --build . ${R_ARGS}
+
```
Comment (dbast) [inline recipes/r-smoothr/build.sh:10]:
```suggestion
#!/bin/bash
export DISABLE_AUTOBREW=1
${R} CMD INSTALL --build . ${R_ARGS}
```

File: recipes/r-smoothr/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/smoothr_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/smoothr/smoothr_{{ version }}.tar.gz
+  sha256: 580b150009934475173a6fc917020bc423934103021a8d3e6b6620d2b93fc5e9
+
+build:
+  skip: true  # [win]
```
Comment (dbast) [inline recipes/r-smoothr/meta.yaml:16]: Its a noarch package as it does not contain compiled code... the windows failures are to be ignored, as the package will be build on linux only after the feedstock conversion.
```suggestion
  noarch: generic
```

File: recipes/r-smoothr/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: covr, knitr, lwgeom, methods, rmarkdown, sp, testthat
```
Comment (dbast) [inline recipes/r-smoothr/meta.yaml:22]:
```suggestion
```

File: recipes/r-smoothr/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    - testthat/
+  commands:
+    - $R -e "library('smoothr')"           # [not win]
+    - $R -f testthat.R                     # [not win]
+    - "\"%R%\" -e \"library('smoothr')\""  # [win]
+
+
+about:
+  home: https://strimas.com/smoothr/, https://github.com/mstrimas/smoothr
+  license: GPL-3.0-or-later
```
Comment (dbast) [inline recipes/r-smoothr/meta.yaml:52]: cran meta data says: # License: GPL-3
```suggestion
  license: GPL-3.0-only
```

**Comment by Rassine** [comment]
@conda-forge/r clould you please be maintainer for this recipe ?

**Review by dbast — APPROVED** [review]

**Comment by dbast** [comment]
@Rassine thanks and welcome to conda-forge!

**Pull request merged**